app/code/core/Mage/Wishlist/Controller/Abstract.php

```diff
@@ -85,7 +85,10 @@ public function allcartAction()
 
         $qtysString = $this->getRequest()->getParam('qty');
         if (isset($qtysString)) {
-            $qtys = array_filter(json_decode($qtysString), '\strlen');
+            $qtys = array_filter(json_decode($qtysString), fn($tmpString) => strlen($tmpString ?? ''));
+            // $qtys = array_filter(json_decode($qtysString), function ($tmpString) { return strlen($tmpString ?? ''); });
+            // $qtys = array_filter(json_decode($qtysString), function ($tmpString) { return $tmpString !== NULL && strlen($tmpString); }); => From Drupal Community
+            // $qtys = @array_filter(json_decode($qtysString), '\strlen');
```

Comment:
```suggestion
            $qtys = array_filter(json_decode($qtysString), fn ($tmpString) => strlen($tmpString ?? ''));
```

this also fixed phpcs

Reply:
Great. Now all greens.